app/controllers/services/service_keys_controller.rb

```diff
@@ -0,0 +1,42 @@
+require 'services/api'
+
+module VCAP::CloudController
+  class ServiceKeysController < RestController::ModelController
+    define_attributes do
+      to_one :service_instance
+      attribute :name, String
+    end
+
+    post path, :create
+    def create
+      @request_attrs = self.class::CreateMessage.decode(body).extract(stringify_keys: true)
```

Comment:
This method is also missing many tests. A good rule of thumb is that commenting out any line of code should result in a test failure. Mostly, I'm thinking that all of the errors that are getting raised need to have associated tests.